src/dynatrace/opentelemetry/metrics/export/_constants.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2022 Dynatrace LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import opentelemetry.sdk.metrics as metrics
+from opentelemetry.sdk.metrics._internal.aggregation import (
+    AggregationTemporality,
+)
+
+_DYNATRACE_TEMPORALITY_PREFERENCE = {
```

Comment:
I guess the Python SDK doesn't have this defined as the other SDKs? Maybe we could consider sending a PR in, so we can deprecate this at some point.